### DISCUSSION TITLE
Getting error while call the ./simulate in terminal stating that illegal hardware instruction

### DISCUSSION THREAD

**Discussion by KishoreKicha14** [google-deepmind/mujoco#542] [General]
Getting error while call the ./simulate in terminal stating that illegal hardware instruction
can anyone help me with it
<img width="623" alt="Screen Shot 2022-10-20 at 12 45 12 PM" src="https://user-images.githubusercontent.com/30365405/197043318-c028a7ff-04a4-4a1a-942c-b48affb63273.png">

**Comment by saran-t** [comment]
The screenshot shows that you're trying to run MuJoCo 2.0. You should've stated that in your issue posting.

MuJoCo 2.0 isn't built for ARM64. Please use MuJoCo 2.1.1 or newer.

**Comment by KishoreKicha14** [comment]
Thanks
